code/modules/projectiles/ammo_datums.dm

```diff
@@ -2545,19 +2547,21 @@ datum/ammo/bullet/revolver/tp44
 	hud_state = "flame"
 	hud_state_empty = "flame_empty"
 	damage_type = BURN
-	flags_ammo_behavior = AMMO_INCENDIARY|AMMO_IGNORE_ARMOR|AMMO_FLAME|AMMO_EXPLOSIVE
+	flags_ammo_behavior = AMMO_INCENDIARY|AMMO_FLAME|AMMO_EXPLOSIVE
 	armor_type = "fire"
 	max_range = 7
-	damage = 0
+	damage = 31
+	damage_falloff = 0
+	incendiary_strength = 30 //Firestacks cap at 20, but that's after armor.
 	bullet_color = LIGHT_COLOR_FIRE
 	var/fire_color = "red"
-	var/burnlevel = 31
 	var/burntime = 17
+	var/burnlevel = 31
```

Comment:
You are doubling impact damage of projectile flamers, aswell as fire stack application. Flamer ammo's fire being created on a tile due to e.g. a projectile flamer's flame hitting a target already iterates through any mobs on it and triggers flamer_fire_act on them, which deals damage and ignites targets. So, there is no such thing as it "relying on the target to be moving", if you actually hit it.

Reply:
Turfs igniting mobs seem to be processed every tick or something. This was something I noticed when I made the tx54. I could ignite the turf I was standing on, and if I walked off it quickly enough, I wouldn't ignite.

That said, people also see this sort of issue due to the fact that movement is instant, but the visual effect of your sprite moving from one turf to another is not. So it can look like you're hitting a turf with a mob on it, but they've actually already moved.